notebooks/text_models/solutions/rnn_encoder_decoder.ipynb

```diff
@@ -4,25 +4,33 @@
    "cell_type": "markdown",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #15.        reset_after=False,
</pre><p>I'm curious about this. Why is it <code>reset_after</code> added here?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/719/discussion/#comment-2431166177'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>

Reply:
<p>The document seems to be saying <code>reset_after=True</code> is GPU compatible.</p><p>But when I tested the <code>True</code> option I saw another weird Graph error, which forces us to set <code>run_eagerly=True</code> in compile (again!), and then another issue in prediction (batch dimension disappeared from decoder state)... </p><p> </p><p>I'm okay with that <code>reset_after=False</code> option since debugging them may be a bit tricky, but could update the markdown description about GPU? Thanks!</p>